### ISSUE TITLE
2i2c Access - Iteration & Documentation

### ISSUE THREAD

**Issue by jules32** [Openscapes/how_we_work#329]
Hi Andy! 

We can use this issue to help overall progress/coordination. This will start off by reading and getting familiar with the following documentation and Issues, with an eye towards having all updated policies and process documented in the Cloud Cookbook (likely in the [Leading Workshops](https://nasa-openscapes.github.io/earthdata-cloud-cookbook/leading-workshops/) section). We can track specific issues in the NASA-Openscapes GitHub organization. 

**Policy documentation and issues:**  

- [x] https://github.com/NASA-Openscapes/2i2cAccessPolicies
  - [x] Also: FYI we have Discussions in this repo: https://github.com/NASA-Openscapes/2i2cAccessPolicies/discussions
- [x] [Cookbook: Leading Workshops](https://nasa-openscapes.github.io/earthdata-cloud-cookbook/leading-workshops/) and how to add people to 2i2c
  - [x] Note new process with `nasa-openscapes-workshops` repo where we will have a separate GitHub Team for each workshop. We can bulk-add GitHub usernames with [kyber](https://openscapes.github.io/kyber/); note this is something that Stef/Julie/Erin currently do for Champions and we're just starting to use for NASA JupyterHub; but hasn't been taught/established as practice for Mentors
  - [x] https://github.com/NASA-Openscapes/workshop-planning/issues/9#issuecomment-1988968323
  - [x] https://github.com/orgs/NASA-Openscapes/projects/7/views/1?pane=issue&itemId=50591614 - also relevant for organization/design: this Leading Workshops section will ultimately have other pages than 2i2c Access
- [x] [2i2c Hackday conversations/notes](https://docs.google.com/document/d/1Rd5vk1B19ldzv2itlHvRXVngcyQJTzmt-SF4mxHV8qM/edit#heading=h.qen4i7qn5dx9) 

**Moving towards automating more workshop processes:** 

- [x] [NASA-OpenscapesCloudWorkshopsRegistry](https://docs.google.com/spreadsheets/d/1Sg4fMDGxMLI0p5cgK5xyamFJpKC6NyNneqXLDReSKcg/edit#gid=695033382) is where we're starting to track all workshops. The idea is that we can follow a similar process we've developed for our Champions Cohorts, using the `kyber` R package. 
- [x] Review kyber: https://openscapes.github.io/kyber/. You'll get a sense of this in the "How We Work" calls starting next week too. We already use kyber to bulk-add github usernames to the GitHub API

**Comment by jules32** [comment]
We've scheduled a 2 hour coworking for April 10 where we can tackle some of these things together once you're a bit more familiar with the things above

**Comment by ateucher** [comment]
This issue also seems relevant here: https://github.com/NASA-Openscapes/earthdata-cloud-cookbook/issues/316

**Comment by ateucher** [comment]
A thought about the =`[ChampionsAccess-2i2c](https://github.com/orgs/nasa-openscapes-workshops/teams/championsaccess-2i2c)` team... is the intention to have one team, and on-board and off-board members at the start/end of each cohort? Or do we want a team for each cohort? I wonder where the cost/benefit falls of administering multiple teams vs potential overlap in access between one team and the next?

**Comment by jules32** [comment]
Progress today: @ateucher created a draft PR to 2i2c/infrastructure to update the teams access: https://github.com/2i2c-org/infrastructure/pull/3930

We stopped before adding mentors (using kyber) to nasa-openscapes-workshops:: AdminTeam, thinking that we will do that for upcoming workshops when we onboard specific mentors to this process (with some of this documentation in the Cookbook)

**Comment by ateucher** [comment]
WE also created a new (empty) team for the 2024 champions cohort `nasa-openscapes-workshops:champions-access-2i2c-2024`

**Comment by jules32** [comment]
Hi Andy! Great job today with presenting [these slides](https://docs.google.com/presentation/d/1iaqP8Hc4sEYBJnwftKFRhVGTaQPu2uLvQboQB-Dbeew/edit#slide=id.p) with the Champions Cohort. I think we're ready to add to Cookbook: 

Could you please: 
- [ ] Move 2i2cAccess policies to [Cookbook Policies and Usage page](https://nasa-openscapes.github.io/earthdata-cloud-cookbook/policies-usage/) 
  - [ ] delete README text and point to Cookbook page
  - [ ] move the current "AWS Cost Explorer" stuff to a new page (maybe exploring-costs.md) and include a link to your slides from today up at the top?

**Comment by stefaniebutland** [comment]
@jules32 @ateucher What's the status of this issue? 
I couldn't resist clearing a few "in progress" items; moved some aspirational things to Backlog.

**Comment by jules32** [comment]
Thanks Stef, Andy has a closer sense of things and we can chat about it in the next weeks and reassess!

**Comment by jules32** [comment]
@ateucher I think actually we've done a lot of these things; maybe it is ok to close?

**Comment by ateucher** [comment]
Yes I think we can close this - thanks @jules32 and @stefaniebutland for raising it! We actually ended up moving this material to openscapes.cloud: https://openscapes.cloud/github-access.html and https://openscapes.cloud/usage-reporting.html
